static/js/compose.js

```diff
@@ -745,6 +743,9 @@ $(function () {
 
     $("#compose").on("click", "#attach_files", function (e) {
         e.preventDefault();
+        if (clone_file_input === undefined) {
+            clone_file_input = $('#file_input').clone(true);
+        }
```

Comment:
hmm, just looking at where you added this change, does this code work in the case of e.g. dragging files into compose?